.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- This template is for Devs to give QA details before moving the issue To-Test -->
+
+<!-- Link to the issue -->
+
+### Root cause
+<!-- Define findings related to the feature or bug issue. -->
+
+### What was fixed, or what changes have occurred
+<!-- Include information of the changes, including collateral areas which have been affected by this PR as requirement or for convenience. -->
+
+### Areas or cases that should be tested
+<!-- Areas that should be tested can include Airgap checks, Rancher upgrades, K8s upgrade, etc. -->
+
+### What areas could experience regressions?
+<!-- Create a detailed list of areas to be analyzed which may be affected by the changes, which would require a prior research to avoid regressions. -->
+
+### Are the repro steps accurate/minimal?
```

Comment:
I feel this is repetitive. I don't see the purpose of this question. This could be just a comment in `areas/cases to be tested`

Reply:
I think originally the aim was to distinguish actual QA steps and secondary checks related to a regression. However I felt the same of you when I tried to use it.